.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+updates:
+  # Monitor Github Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "10:00"
+    commit-message:
+      prefix: "chore"
+      include: "scope"
+    open-pull-requests-limit: 10
+  # Monitor Python test dependencies
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "10:00"
+    commit-message:
+      prefix: "chore"
+      include: "scope"
```

Comment:
Same here, we could have like `prefix`: `requirements`
I also suggest adding the `prefix-development`: `requirements dev`

Reply:
Done 👍